examples/game/lib/game_demo.dart

```diff
@@ -5,6 +5,7 @@ import 'dart:math' as math;
 import 'dart:sky' as sky;
 
 import 'package:sky/rendering/object.dart';
+import 'package:sky/painting/text_style.dart';
 import 'package:sky/widgets/framework.dart';
```

Comment:
You should import:

import 'package:sky/rendering.dart';
import 'package:sky/widgets.dart';

That will give you text_style for free.